index.js

```diff
@@ -28,7 +28,7 @@ function Peer (opts) {
 
   stream.Duplex.call(self, opts)
 
-  opts.channelName ? self.channelName = opts.channelName : randombytes(20).toString('hex')
+  opts.channelName ? self.channelName = opts.channelName : self.channelName = randombytes(20).toString('hex')
```

Comment:
This is better written as `self.channelName = opts.channelName || randombytes(20).toString('hex')`